### DISCUSSION TITLE
Installing NGINX on UDMP

### DISCUSSION THREAD

**Discussion by Mordrin90** [unifi-utilities/unifi-common#640] [General]
Been trying to setup a docker within my UDMP running 4.1.9. I've been following a pretty incomplete guide here;
https://lickthesalt.com/guides/nginx-proxy-manager-udmp/
It's not up to date and I've been battling this process for the past two days.
I'm finally at the point where I'm attempting to run the docker command to download and install and I'm getting an out of space error.

no space left on device
Error: Error committing the finished image: error adding layer with blob "sha256:3eae342e55dd703671e5c462aa306e383ca32a1fda17241532bfdbcf98ccc088": error creating layer with ID "2691b6ca587e68dde214b9a89ed0626e2e8a12bf7077b268d035809f2ac62533": no space left on device

I'm unsure how to resolve this as this whole process form what I can see has not created a partition with X size for me to change to allow additional space for the docker to download and install.

Any help would be great.

**Comment by m4n63** [comment]
My understanding is that podman is not supported anymore, and you need to use this approach instead
https://github.com/unifi-utilities/unifios-utilities/tree/main/nspawn-container
